File: python/nowy program.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
#
#  nowy program.py
 
def pobierzOceny():
    oceny = []
    ocena = int(input('Podaj ocenę: '))
    while ocena > 0:
        if ocena < 7:
            oceny.append(ocena)
        else:
            print('Nie ma takiej !')
        ocena = int(input('Podaj ocenę: '))
    return oceny
    
def liczSrednia(oceny):
    print(sum(oceny) / len(oceny))

def main(args):
    oceny = pobierzOceny()
    liczSrednia(oceny)
    return 0

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))

File: c++/nowyprogram.cpp
/*
 * nowyprogram.cpp
 */


#include <iostream>
using namespace std;

int main(int argc, char **argv)
{
    float ocena, liczbaocen, suma;
    suma = 0;
    cout << "Podaj liczbę ocen: ";
    cin >> liczbaocen;
    while (liczbaocen < 1) {
        cout "podaj liczbę ocen: ";
        cin >> liczbaocen;
    }
	for(5) {
            cout << "Podaj ocenę: ";
            cin >> ocena;
            while (ocena > 6) {
                cout << "Podaj prawidłową ocenę: " ;
                cin >> ocena;
                suma = suma + ocena;
        }
    }
	return 0;
}

